lib/aruba/api.rb

```diff
@@ -729,6 +735,61 @@ def io_wait
       @aruba_io_wait_seconds || DEFAULT_IO_WAIT_SECONDS
     end
 
+    DEFAULT_ROOT_DIRECTORY = Dir.getwd
+
+    # The root directory of aruba
+    def root_directory
+      @aruba__root_directory ||= DEFAULT_ROOT_DIRECTORY
+    end
+
+    DEFAULT_FIXTURES_PATH_PREFIX = '%/'
+
+    def fixtures_path_prefix
```

Comment:
Making the fixture prefix character configurable via `@aruba_fixture_path_prefix`. To be honest I think it should not be configurable; just like `~` referring to HOME directory is not configurable.